PerimeterXModule/DataContracts/Requests/Additional.cs

```diff
@@ -28,5 +28,18 @@ public class Additional
 
 		[DataMember(Name = "px_cookie_hmac")]
 		public string PxCookieHMAC;
+
+		[DataMember(Name = "cookie_origin")]
+		public CookieOrigin CookieOrigin;
+
+		[DataMember(Name = "original_uuid")]
+		public string OriginalUUID;
+
+		[DataMember(Name = "original_token_error")]
+		public string OriginalTokenError;
+
+		[DataMember(Name = "decoded_original_token")]
+		public object DecodedOriginalToken;
+
```

Comment:
What about passing the `original_token` itself? ([node example](https://github.com/PerimeterX/perimeterx-node-core/blob/b9b1569697d9de6c42388443589be1b847198903/lib/pxapi.js#L71))

Reply:
the token itself will be populated on the headers as header_original_token by the collector

Reply:
So let's please make sure that the design doc is updated to reflect this approach. 
As far as I understand the doc, you should _retrieve the original token_ -> _set it on the context_ -> .... (do a bunch of other logic ) ... -> when sending the risk you should set the *original_uuid + original_token_error (if exists) +* _**original_token**_ (if *original_token_error* exists) on the request details.